examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,487 @@
+mapdl.nsel("R", "LOC", "Y", 0)  # SELECT NODES AT THETA = 0 FROM THE SELECTED SET
+mapdl.dsym("SYMM", "Y", 1)  # APPLY SYMMETRY BOUNDARY CONDITIONS
+mapdl.nsel("S", "NODE", "", 101, 212)  # SELECT ONLY NODES IN SOLID185 AND SHELL181 MODELS
+mapdl.nsel("R", "LOC", "Y", theta)  # SELECT NODES AT THETA FROM THE SELECTED SET
+
+mapdl.dsym("SYMM", "Y", 1)  # APPLY SYMMETRY BOUNDARY CONDITIONS
+
+mapdl.nsel("ALL")
+mapdl.nsel("R", "LOC", "Z", 0)  # SELECT ONLY NODES AT Z = 0
+
+mapdl.d("ALL", "UZ", 0)  # CONSTRAIN BOTTOM NODES IN Z DIRECTION
+
+mapdl.nsel("ALL")
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system. Print the solver output.
+
+# Start solution presederu
+mapdl.run("/SOLU")
+
+# Define solution function.
+def solution(deflect=None):
+    mapdl.nsel("R", "LOC", "Z", 10)
+    mapdl.d(node="ALL", lab="UZ", value=deflect)
+    mapdl.nsel("ALL")
+    mapdl.solve()
+
+
+# Run each load step to reproduce needed deflection subsequently.
+# Load Step 1
+solution(deflect=defl_ls1)
+
+# Load Step 2
+solution(deflect=defl_ls2)
+
+# Load Step 3
+solution(deflect=defl_ls3)
+mapdl.finish()
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing.
+
+# Enter the post-processing routine
+mapdl.run("/OUT,")
+mapdl.post1()
+
+
+###############################################################################
+# Define loads.
+# ~~~~~~~~~~~~~
+# Set up the function.
+
+def getload():
+    ####
+    # Select the nodes in the PIPE288 element model.
+    mapdl.nsel("S", "NODE", "", 1, 2)
+    mapdl.nsel("R", "LOC", "Z", 0)
+    mapdl.run("/OUT,SCRATCH")
+
+    # Sum the nodal force contributions of elements.
+    mapdl.fsum()
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_288 = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    ####
+    # Select the nodes in the SOLID185 element model.
+    mapdl.nsel("S", "NODE", "", 101, 118)
+    mapdl.nsel("R", "LOC", "Z", 0)
+    mapdl.fsum()
+
+    # Get the force value of the simplified model.
+    load_185_theta = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_185 = load_185_theta * 360 / theta
+    mapdl.run("*STATUS,LOAD")
+
+    ####
+    # Select the nodes in the SHELL181 element model.
+    mapdl.nsel("S", "NODE", "", 201, 212)  # SELECT NODES IN SHELL181 MODEL
+    mapdl.nsel("R", "LOC", "Z", 0)
+
+    # Sum the nodal force contributions of elements.
+    mapdl.fsum()
+    mapdl.run("/OUT,")
+
+    # Get the force value of the simplified model.
+    load_181_theta = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_181 = load_181_theta * 360 / theta
+    mapdl.run("*STATUS,LOAD")
```

Comment:
Is this necessary?

Reply:
`mapdl.run("/OUT,"), 
mapdl.run("*STATUS, LOAD")`
 
  I know that they are not needed, 
  but I have left them just in vm7 to show that we can use them if we want.
  As our task in VMs to show not just the academic side but also show maximum functionality.

Reply:
Removed all.